test/test_binary_ufuncs.py

```diff
@@ -1048,6 +1048,20 @@ def test_maximum_minimum_complex(self, device, dtypes):
             with self.assertRaisesRegex(RuntimeError, 'does not support complex inputs'):
                 torch_op(torch.ones(1, device=device, dtype=dtypes[1]),
                          torch.ones(1, device=device, dtype=dtypes[0]))
+    @onlyCUDA
+    def test_maximum_minimum_device_mismatch(self, device):
```

Comment:
please add a test operating on cuda tensor and cpu scalar (`a=torch.tensor(1), b=torch.tensor((3,0,4), device=device`) - it should not raise an error.